topics/remote/index.md

```diff
@@ -0,0 +1,26 @@
+---
+aliases: [remote-work, remote-collaboration]
+created_by: "@rendred Tech Headhunter"
```

Comment:
As far as I know, quotes around properties are not needed. Haven't seen them in any other topic file so far 🤔